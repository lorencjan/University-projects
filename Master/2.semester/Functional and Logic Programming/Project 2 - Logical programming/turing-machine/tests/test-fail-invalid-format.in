S a S Q
S b S R
S   F  
aaaaaabbaaaabbbbbbbbbb
