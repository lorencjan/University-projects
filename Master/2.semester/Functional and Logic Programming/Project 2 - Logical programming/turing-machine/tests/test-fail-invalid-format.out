Input error: Invalid format of the TM.